queries.yaml

```diff
@@ -2009,16 +2036,26 @@ post_process:
       start_zoom: 8
       end_zoom: 15
       # pixels (256px nominal) to require per letter in the name. if lines are shorter than this,
-      # the name gets dropped. name pairs (e.g: :left and :right) get dropped if either are longer
-      # than pixels_per_letter. larger values mean fewer lines are eligible for labelling.
+      # the name gets dropped. larger values mean fewer lines are eligible for labelling.
       pixels_per_letter: 11
 
+  # drop labels on roads which are too short to render
+  # but with more flexibility for range of style text sizes
+  - fn: vectordatasource.transform.drop_names_on_short_boundaries
+    params:
+      source_layer: roads
+      start_zoom: 15
+      end_zoom: 17
+      # pixels (256px nominal) to require per letter in the name. if lines are shorter than this,
+      # the name gets dropped. larger values mean fewer lines are eligible for labelling.
+      pixels_per_letter: 8
```

Comment:
For 15-8878-13732, look at the street formerly known as Lafayette Street, which combined with property dropping above, and the pixels_per_letter going to 8 here, should yield labels restored versus the 1st draft of this PR:

<img width="1205" alt="image" src="https://user-images.githubusercontent.com/853051/153136959-2b58aa1f-65bc-4596-918f-8f3b7b48025d.png">

Reply:
8 is still too big, still not seeing E. Kennedy labeled

Reply:
82 / 15 = 5 pixels per character

Reply:
Success! 

<img width="1157" alt="image" src="https://user-images.githubusercontent.com/853051/153696283-eee39b72-c3cd-406b-8d82-d71958da4ce0.png">